src/glum/_glm.py

```diff
@@ -327,17 +327,17 @@ def _standardize(
     X, col_means, col_stds = X.standardize(sample_weight, center_predictors, True)
 
     if col_stds is not None:
+        penalty_mult = _one_over_var_inf_to_val(col_stds, 1.0)
```

Comment:
Now that we use `penalty_mult` in various places, maybe a different name would be appropriate (such as `col_stds_inv`) - at least I was confused by the term "penalty".